src/main/dao/TransactionsDaoCsvImpl.java

```diff
@@ -0,0 +1,56 @@
+package main.dao;
+
+import main.model.ProductTransaction;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class TransactionsDaoCsvImpl implements TransactionsDao {
```

Comment:
Dao stands by Data Acess Object and typically indicates that we are working with Database. 

Better to rename.